Cartesian_kernel/include/CGAL/constructions/kernel_ftC2.h

```diff
@@ -283,9 +283,17 @@ line_get_pointC2(const FT &a, const FT &b, const FT &c, const FT &i,
       // The test `!boost::is_integral<FT>::value` is there to avoid
       // that `a != 0` is tested on anything but integral types, for
       // performance reasons.
-      CGAL_assume(!boost::is_integral<FT>::value || a != FT(0));
-
+      //CGAL_assume(!boost::is_integral<FT>::value || a != FT(0));
+    
+    //the previous code doesn't work, the warning is still there. Definitely
+    //mute it using pragma.
+#ifdef _MSC_VER
+#pragma warning(disable:4723) 
+#endif _MSC_VER
```

Comment:
This line `#endif _MSC_VER` will also produce a warning, because one cannot have anything after `#endif`. Maybe you wanted a comment:
```suggestion
#endif // _MSC_VER
```

And same issue below.